.github/workflows/integration_tests.yml

```diff
@@ -429,11 +438,27 @@ jobs:
 
       - name: Test Continuous Delivery (with sockshop)
         id: test_continuous_delivery
-        if: env.RUN_CONTINUOUS_DELIVERY_TEST == 'true' # run only if variable is set
+        if: (env.RUN_CONTINUOUS_DELIVERY_TEST == 'true') && env.REMOTE_EXECUTION_PLANE != 'true'
```

Comment:
Not sure if you also need to have parentheses around the expressions, e.g.:
```
        if: (env.RUN_CONTINUOUS_DELIVERY_TEST == 'true') && (env.REMOTE_EXECUTION_PLANE != 'true')
```
I've seen it in many examples, though I don't think parentheses is necessary here - you can leave it as it is